Automattic/jetpack
# VideoPress: check if it's an auto-generated chapters file

This PR checks if the video has a not auto-generated chapter .vtt file no avoid overwriting it.

Fixes https://github.com/Automattic/jetpack/issues/26263

#### Changes proposed in this Pull Request:

* VideoPress: check if chapters .vtt file exists

#### Other information:

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to Block editor
* Since uploading a vtt file is not migrated to v6 yet, use a `core/video` extended video block (Jetpack plugin) to upload the file
<img width="357" alt="image" src="https://user-images.githubusercontent.com/77539/203293439-e4dce7ca-7146-4c64-a782-8343a45fb288.png">

<img width="682" alt="image" src="https://user-images.githubusercontent.com/77539/203294135-4a15292e-1a2e-476d-8e5c-37547b53efa8.png">

<img width="695" alt="image" src="https://user-images.githubusercontent.com/77539/203297166-c869fa4a-286e-4c78-81b6-c4dac1388d40.png">

* Save the post
* Hard refresh
* Confirm the video has chapters
<img width="570" alt="Screen Shot 2022-11-22 at 07 56 50" src="https://user-images.githubusercontent.com/77539/203297346-2fbbaa2a-55ab-4081-b122-0aa351fd5023.png">

* Transform the block to v6
* Open block sidebar
* Confirm it shows a notice about the vtt chapters file

<img width="999" alt="Screen Shot 2022-11-22 at 07 59 32" src="https://user-images.githubusercontent.com/77539/203297616-803985a3-19cc-4f50-b8f0-3d7c4f8978b2.png">

* Confirm even when the description has chapters, it does not overwrite the current one.

* Create a new block with a video (ensure it doesn't have chapters)
* Confirm you do not see the notice
* Confirm you can add chapters
